styles.css

```diff
@@ -316,7 +316,7 @@ body {
 
```

Comment:
- [ ] [OPTIONAL]The design of your project is great 👍  but I noticed that there is some margin` or `padding` on the right and left side of the body elements, it would be better if you fix this error 😃 

![image](https://user-images.githubusercontent.com/63915024/193931355-18702466-78cf-41e9-87d2-8399d232922d.png)